src/pages/index.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+
+import Home from 'pages/index.page';
+
+describe('Home', () => {
+  it('renders a heading', () => {
+    render(<Home />);
+
+    const heading = screen.getByRole('heading', {
+      name: /welcome to next\.js!/i,
+    });
+
+    expect(heading).toBeInTheDocument();
```

Comment:
maybe we could use `toBeVisible()` instead as it [makes sure](https://testing-library.com/docs/dom-testing-library/api-configuration/) the element does not have css properties like `display: none` or `visibility: hidden`?